easybuild/easyconfigs/r/rootpy/rootpy-1.0.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonPackage'
+
+name = 'rootpy'
+version = '1.0.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.rootpy.org/'
+
+description = """The rootpy project is a community-driven initiative aiming to
+provide a more pythonic interface with ROOT on top of the existing
+PyROOT bindings. Given Python’s reflective and dynamic nature, rootpy
+also aims to improve ROOT design flaws and supplement existing ROOT
+functionality. The scientific Python community also offers a multitude
+of powerful packages such as SciPy, NumPy, matplotlib, scikit-learn,
+and PyTables, but a suitable interface between them and ROOT has been
+lacking. rootpy provides the interfaces and conversion mechanisms
+required to liberate your data and to take advantage of these
+alternatives if needed."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/rootpy/%(namelower)s/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['7d85fc8fc4ff96a833fa593be401aa4836b755757b518d23d4c00d7a105ce27e']
```

Comment:
I got a tarball for rootpy v1.0.1 with the following checksum. Could you verify it?
```
f7e2bec0c2fb6837c97f5060abe0a015243d1130102a0f38807f9de0f7825de5  1.0.1.tar.gz
```

Reply:
```suggestion
checksums = ['f7e2bec0c2fb6837c97f5060abe0a015243d1130102a0f38807f9de0f7825de5']
```